dashboard/src/components/dashboard/AccountMenu.tsx

```diff
@@ -23,21 +21,19 @@ function AccountMenuContent({
   const user = useUserData();
   const router = useRouter();
   const [clicked, setClicked] = useState(false);
-  const { setWorkspaceContext } = useWorkspaceContext();
-  const { setUserContext } = useUserDataContext();
   const { handleClose } = useDropdown();
+  const client = useApolloClient();
 
   return (
-    <div className="relative grid w-account grid-flow-row gap-5 p-6">
+    <div className="relative grid grid-flow-row gap-5 p-6 w-account">
       <Button
         variant="borderless"
         color="secondary"
-        className="absolute top-6 right-4 grid grid-flow-col items-center gap-1 self-start font-medium"
+        className="absolute grid items-center self-start grid-flow-col gap-1 font-medium top-6 right-4"
```

Comment:
What tool are you using for this?

Reply:
I have the `Headwind` VS code extension, which I think is sorting this classes (in a kind of arbitrary way? none are alphabetically) I did think that `prettier-plugin-tailwindcss` overrided this in our `precommit`

Reply:
I have removed Headwind as it conflicted with `prettier-plugin-tailwindcss`. I think we can roll with the Prettier plugin only, because that's IDE agnostic.

Reply:
:+1: I have reseted this branch 2 commits prior and force pushed so we don't see this changes (also disabled the Headwind extension)